src/window_savefile.cpp

```diff
@@ -134,12 +131,17 @@ void Window_SaveFile::Refresh() {
 
 	int hx = Font::Default()->GetSize(hp_short).width;
 	out.str("");
-	out << std::setw(Player::IsRPG2k3() ? 4 : 3) << std::setfill(' ') << hero_hp;
+	out << std::setw(Player::IsRPG2k3() ? 4 : 3) << std::setfill(' ') << data.hero_hp;
 	contents->TextDraw(46 + hx, 32 + 2, fc, out.str());
 
-	for (int i = 0; i < 4 && (size_t) i < party.size(); i++) {
-		DrawFace(party[i].second, party[i].first, 92 + i * 56, 0, false);
-	}
+	int i = 0;
```

Comment:
...is this a micro optimization? :grinning:

Could even save the `++i` lines and use `i++` directly in `DrawFace`.

Reply:
Removing the vector was a micro optimization of sorts, but it's more about convenience of reusing the `SaveTitle` struct.

Added `i++`, much cleaner that way thanks!